### ISSUE TITLE
runtime cosock?? error in Zigbee Sensor driver

### ISSUE THREAD

**Issue by veonua** [SmartThingsCommunity/SmartThingsEdgeDrivers#12]
```
2021-08-31T22:31:44.574557453+00:00 TRACE Zigbee Motion Sensor  Received event with handler zigbee
2021-08-31T22:31:44.595245120+00:00 INFO Zigbee Motion Sensor  <ZigbeeDevice: 25931f2c-e1d6-4a73-9786-2209b583f13a [0x03BD] (Aqara Motion Illuminance Sensor)> received Zigbee message: < ZigbeeMessageRx || type: 0x00, < AddressHeader || src_addr: 0x03BD, src_endpoint: 0x01, dest_addr: 0x0000, dest_endpoint: 0x01, profile: 0x0104, cluster: IlluminanceMeasurement >, lqi: 0x8C, rssi: -65, body_length: 0x0008, < ZCLMessageBody || < ZCLHeader || frame_ctrl: 0x18, seqno: 0x3E, ZCLCommandId: 0x0A >, < ReportAttribute || < AttributeRecord || AttributeId: 0x0000, DataType: Uint16, MeasuredValue: 0x0012 > > > >
```
2021-08-31T22:31:44.625621787+00:00 FATAL Zigbee Motion Sensor  Lua: runtime error: [string "cosock"]:241: **[string "socket"]**:64: attempt to call a nil value (field 'warning')
stack traceback:
	[string "socket"]:64: in local 'cb'
	[string "st.driver"]:741: in function <[string "st.driver"]:700>
stack traceback:
	[C]: in function 'error'
	[string "cosock"]:241: in field 'run'
	[string "st.driver"]:749: in method 'run'
	[string "init.lua"]:43: in main chunk

**Comment by FreeMasen** [comment]
Thanks for opening this issue! I am looking through the code and not seeing any references to a field named `warning` so far but I am going to keep looking. 

Looking at the traceback, it seems like the error _might_ be coming from inside of a callback, would you mind confirming for me that your code doesn't have reference to a property `warning`?

**Comment by veonua** [comment]
my code does not contain any inclusions of the string "warning".

**Comment by posborne** [comment]
Thanks again for the report @veonua, there was some confusion at first as the change was on the release branch for the 0.39 beta and not upstreamed internally as of yet.  We are see the problem and the team should have a fix included in the next update to the 0.39 beta.

Thanks for your patience and for reporting this issue and helping us smoke out some of these issues during the beta.

**Comment by veonua** [comment]
seems to be fixed now
